docs/tables/oci_core_drg.md

```diff
@@ -0,0 +1,44 @@
+# Table: oci_core_drg
+
+ A dynamic routing gateway (DRG), which is a virtual router that provides a path for private network traffic between your VCN and existing network.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  display_name,
+  id,
+  lifecycle_state,
+  time_created
+from
+  oci_core_drg;
+```
+
+
+### List unavailable dynamic routing gateways
+
+```sql
+select
+  display_name,
+  id,
+  lifecycle_state
+from
+  oci_core_drg
+where
+  lifecycle_state <> 'AVAILABLE';
+```
+
+
+### Count of dynamic routing gateways per region
```

Comment:
Is this query useful?

Reply:
can we write a query to get the details of vcn attached to drg?

Reply:
we don't get any vcn info in the response.